File: client/src/app/shared/model/licenses/license.rules.ts
export enum LicenseName {
    FREE = 'free',
    OSINT_BASIC = 'osint_basic',
    OSINT_ADVANCED = 'osint_advanced',
    PENTESTER = 'pentester',
    DATA_MANAGER = 'data_manager',
    ENTERPRISE = 'enterprise'
}


export interface LicenseRuleModel {
    modules: string[] | 'all';
    cti_graph: boolean;
    mapping: boolean;
    scanning: boolean;
    data_manager: boolean;
}

File: backend/orion/api/interactive/auth_manager/rules/license_rules.py
LICENSE_RULES = {
    "free":{
        "modules": ["general"],
        "cti_graph": False,
        "mapping": False,
        "scanning": False,
        "admin": False,
    },
    "osint_basic": {
        "modules": ["general", "breach", "exploit", "discussion", "defacement", "social", "feed", "dumps"],
        "cti_graph": False,
        "mapping": False,
        "scanning": False,
        "admin": False,
    },
    "osint_advanced": {
        "modules": ["general", "breach", "exploit", "discussion", "defacement", "social", "feed", "dumps", "stealer_logs"],
        "cti_graph": True,
        "mapping": True,
        "scanning": False,
        "admin": False,
    },
    "pentester": {
        "modules": [],
        "scanning": True,
        "cti_graph": False,
        "admin": False,
    },
    "admin": {
        "modules": [],
        "admin": True,
        "scanning": False,
        "cti_graph": False,
    },
    "enterprise": {
        "modules": "all",
        "scanning": True,
        "cti_graph": True,
        "mapping": True,
        "admin": True,
    }
}
